### DISCUSSION TITLE
crashes after runnning 'npm run build'

### DISCUSSION THREAD

**Discussion by tnbossi123** [squidfunk/mkdocs-material#3655] [Q&A]
![image](https://user-images.githubusercontent.com/36259093/156356173-e564798a-9047-49b7-ac77-73830284faf1.png)
![image](https://user-images.githubusercontent.com/36259093/156356223-ab6d4181-9ada-495a-bba0-c5037e1a565c.png)

I have an issue when I run 'npm run build', it seems like the script changes/ subtitutes the local path to all the css and js to global ones. This results to that mkdocs cannot locate the files. Is there something I do wrong or how can I fix this problem?

**Comment by squidfunk** [comment]
When you run `npm run build`, content hashes are added to the files, so browsers do not cache them forever. If you restart `mkdocs serve` after `npm run build`, everything is fine.

**Reply by tnbossi123** [reply]
So I have first ran 'mkdocs serve --watch-theme', checked the page and it worked fine. Then i stopped the watch-theme, and ran 'npm run build', after this i tried to run the watch-theme again, and now the issues occure.....

**Reply by squidfunk** [reply]
Are you extending the theme? If so, you should not touch the parts of `base.html` that include `* .js` and `*.css` files of the theme. If you forked the theme, everything should work as advertised. It's the exact same process I use when working on the theme.

**Reply by tnbossi123** [reply]
Well, i want to extend/ change it up a bit, but this just happens with clean clone of this git repository. A friend of mine tested this on Linux, and it worked fine, i may suspect something with windows and npm.

**Reply by squidfunk** [reply]
It's unlikely that this behavior is related to your operating system.

**Reply by tnbossi123** [reply]
What would you recommend for me to do or how to go about it?

**Reply by squidfunk** [reply]
Make sure you have the latest versions of `node` and `npm` installed. As I said, it's the exact process I'm using for development, so if you follow the [theme development guide](https://squidfunk.github.io/mkdocs-material/customization/#theme-development) you should be fine. Hundreds of other users are doing so.
